python/pyspark/sql/functions/builtin.py

```diff
@@ -16303,7 +16303,21 @@ def from_xml(
     >>> df.select(sf.from_xml(df.value, schema).alias("xml")).collect()
     [Row(xml=Row(a=1))]
 
-    Example 2: Parsing XML with :class:`ArrayType` in schema
+    Example 2: Parsing XML with a :class:`StructType` schema
```

Comment:
Thanks for fixing this. Can you please reuse the existing jira #SPARK-45190?

Reply:
sure, I was not aware of that ticket, will also add it to the title